docs/Meadow/Meadow.OS/Networking/Antenna/index.md

```diff
@@ -4,7 +4,7 @@ title: WiFi/Bluetooth Antenna
 subtitle: Understanding and switching between the onboard and external antenna options.
 ---
 
-Both the Meadow development board and production module have an onboard ceramic chip antenna and a U.FL connector for an external antenna for the 2.4GHz WiFi and Bluetooth radio. Additionally, there is an antenna switch for switching between the two; by default, the chip antenna is selected, and you must use the Meadow.OS device API to switch to the external antenna.
+Both the Meadow F7 Micro development board and Core Compute Module have an onboard ceramic chip antenna and a U.FL connector for an external antenna for the 2.4GHz WiFi and Bluetooth radio. Additionally, there is a software antenna switch for toggling between the two antenna; by default, the chip antenna is selected, and you must use the Meadow.OS device API to switch to the external antenna.
```

Comment:
should probably be "F7 Feather" instead of "F7 Micro"

Reply:
Not sure what I was thinking there. Wonder if I copied that name from somewhere else that would need changing too. I'll get it corrected:

Reply:
lol...there's yer problem right there: 49 results
https://github.com/WildernessLabs/Documentation/search?q=F7+Micro